### PR TITLE
Fixing changed pushed without edit

### PR DESCRIPTION
**Pull request by moxiegirl**
Fixing structure of step.

Signed-off-by: Mary Anthony mary@docker.com

### CODE REVIEW

**Comment by moxiegirl** [comment]
ping @jamtur01 @fredlf @SvenDowideit

File: docs/sources/project/work-issue.md [hunk 1]
```diff
@@ -191,8 +191,10 @@ You should pull and rebase frequently as you work.
 
 	Make sure you include your signature.
 
-8. Push any changes to your fork on GitHub, using the `-f` option to
-force the previous change to be overwritten.
+8. Push any changes to your fork on GitHub.
+
+	The rebased rewrote history, so you'll need to use the `-f` or `--force` flag
```
Comment (fredlf) [inline docs/sources/project/work-issue.md]: Typo: Rebase(d)

**Comment by moxiegirl** [comment]
@fredlf thanks for the catch Fred. Fixed.

**Comment by jamtur01** [comment]
LGTM

**Comment by moxiegirl** [comment]
author LGTM

**Comment by fredlf** [comment]
LGTM too, just waiting on Janky.

**Pull request merged**